### ISSUE TITLE
Make alt text accessible

### ISSUE THREAD

**Issue by kensanata** [ReticentJohn/Amaroq#86]
I tried to read the alt text for images I had uploaded with Amaroq and was unable to do it. I hoped that a long press on an image, either in the toot view or in the image view would show the alt text (media_attachments / description).

**Comment by marcondag** [comment]
@ReticentJohn Do you mind if I pick this up and take a stab at it?

**Comment by ReticentJohn** [comment]
All yours @marcondag - thanks for showing an interest in contributing!
